todoApp/src/components/Button.tsx

```diff
@@ -1,11 +1,37 @@
-//Functional component
-import React from "react";
+import React, { useState } from "react";
+import AddTodoForm from "./AddTodoForm";
 import styles from "./Button.module.css";
 
-const Button = () => {
+interface ButtonInterface {
+  onButtonClicked: () => void;
+}
+
+const Button = ({ onButtonClicked }: ButtonInterface) => {
+  const [clicked, setClicked] = useState(false);
+
+  const buttonHandler = () => {
+    setClicked((prevState) => !prevState);
+  };
+  const cancelHandler = (state: boolean) => {
+    setClicked(state);
+  };
+  const submitHandler = () => {
+    onButtonClicked();
+  };
+
   return (
     <>
-      <button className={styles.button}>+</button>
+      {clicked && (
```

Comment:
Replace these 2 conditional renders with a ternary statement.
Also, the component is named as Button but it contains a form as well. So, give it a meaningful name.